app/services/usps_confirmation_uploader.rb

```diff
@@ -1,9 +1,14 @@
 class UspsConfirmationUploader
+  def initialize
+    @now = Time.zone.now
```

Comment:
next time I'd consider making this a default argument so we could inject it cleanly in tests if we wanted:

```
def initialize(now: Time.zone.now)
  @now = now
end
```